packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,307 @@
+   * --------
+   * All properties common to Image and Video Props
+   * @beta
+   */
+  interface VisualMediaProps {
+    /**
+     * @hidden
+     * Lets the developer specify the visualMedia source
+     */
+    sourceProps: GalleryProps | CameraProps;
+    /**
+     * max limit of media allowed to be selected in one go, current max limit is 10.
+     */
+    maxVisualMediaCount: number;
+  }
+  /**
+   *  All properties in ImageProps are optional and have default values in the platform
+   *  Additional properties for image in mobile devices
+   * @beta
+   */
+  export interface ImageProperties extends VisualMediaProps {
+    /**
+     * Optional; indicate if inking on the selected Image is allowed or not
+     * Default value is true
+     */
+    shouldAllowInkingOnImages?: boolean;
+
+    /**
+     * Optional; indicate if putting text stickers on the selected Image is allowed or not
+     * Default value is true
+     */
+    shouldAllowTextStickersOnImages?: boolean;
+
+    /**
+     * Optional; indicate if the filter mode is enabled on the selected image
+     * Default value is false
+     */
+    enableFilterOnImages?: boolean;
+
+    /**
+     * Optional; Lets the developer specify the image output formats, more than one can be specified.
+     * Default value is Image.
+     */
+    imageFormatsAllowedToBeSelected?: ImageOutputFormats[];
+  }
+
+  /**
+   * The required value of the visualMedia files from gallery
+   * @beta
+   */
+  export interface GalleryProps {
+    /**
+     * @hidden
+     * The visualMedia source
+     */
+    source: Source.Gallery;
+  }
+  /**
+   * The required value of the visualMedia files from camera
+   * @beta
+   */
+  export interface CameraProps {
+    /**
+     * @hidden
+     * The visualMedia source
+     */
+    source: Source.Camera;
+    /**
+     * @hidden
+     * Optional; indicate if user is allowed to move between front and back camera
+     * Default value is FrontOrRear
+     */
+    cameraRestriction?: CameraRestriction;
+  }
+
+  /**
+   * Indicate if user is allowed to move between front and back camera or stay in front/back camera only
+   * @beta
+   */
+  export enum CameraRestriction {
+    /** User can move between front and back camera */
+    FrontOrRear = 1,
+    /** User can only use the front camera */
+    FrontOnly = 2,
+    /** User can only use the back camera */
+    RearOnly = 3,
+  }
+
+  /**
+   * Enum for file formats supported
+   * @beta
+   */
+  export enum FileFormat {
+    /** Base64 encoding */
+    Base64 = 'base64',
+    /** File id */
+    ID = 'id',
+  }
+  /**
+   * Specifies the image source
+   * @beta
+   */
+  export enum Source {
+    /** visual media source is camera. */
+    Camera = 1,
+    /** visual media source is gallery. */
+    Gallery = 2,
+  }
+  /**
+   * Specifies the image output formats.
+   * @beta
+   */
+  export enum ImageOutputFormats {
+    /** Outputs image.  */
+    Image = 1,
+    /** Outputs pdf. */
+    PDF = 2,
```

Comment:
Sorry if I missed this, are we planning to export the result images as PDF also or just Image?

Reply:
I think it is ok to leave the PDF option here for now?